### ISSUE TITLE
[a11y] Mouse scroll up is not working on VPN Tips & Tricks page after keyboard nav to 3rd quick tip

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#4544]
**Mozilla VPN affected versions:**

* 2.10.0 (2.202209300115);

**Platform tested:**

* Windows 10;

**Prerequisites:**

* Be logged in with an account with subscription for Mozilla VPN;

**Steps to reproduce:**

1. Open the VPN app 
1. Go to Settings - VPN Tips and Tricks option;
1. Continue navigation using the keyboard;
1. Go to the 3rd quick tip;
1. Scroll the page up using the mouse;
1. Observe the behavior;

**Expected results:**

* Both keyboard and mouse navigation works as expected;

**Actual results:**

* After keyboard navigation until the 3rd tip, user cannot scroll up until the top of the screen is displayed;

**Notes:**

* Quitting the tips and tricks screen fixes the issue;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-2956)
┆Reporter: Valentina Virlics

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this on 2.14.0 (2.202303071700) while using Windows 10 and the mouse scroll works as expected after using keyboard navigation.

But, the keyboard navigation is not visible because the page is not scrolled down. Should we leave this open to track this or log another issue? Matt Lichtenstein 

Thank you!

**Comment by data-sync-user** [comment]
➤ Matt Lichtenstein commented:

Valentina Virlics the bug you are describing sounds like VPN-2958, right? If so, let’s close this one and track keyboard nav in that one

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Matt Lichtenstein True that!

Closing this as fixed.
